pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -2595,7 +2595,7 @@ protected void internalSetOffloadPolicies(AsyncResponse asyncResponse, OffloadPo
             final String path = path(POLICIES, namespaceName.toString());
             byte[] content = globalZk().getData(path, null, nodeStat);
             Policies policies = jsonMapper().readValue(content, Policies.class);
-
+            // there is possibility that `offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis()` is null.
```

Comment:
Shall we need to add this comment? look we already have a null check for deletion lag.

Reply:
OK